app/views/guidance/hesa_register_data_mapping.html.erb

```diff
@@ -0,0 +1,49 @@
+<%= render PageTitle::View.new(text: "Hesa register data mapping") %>
```

Comment:
```suggestion
<%= render PageTitle::View.new(text: "Check the data mapping between HESA and Register trainee teachers") %>
```

Ideally the title would include the partial for the tab. Eg `Check the data mapping between HESA and Register trainee teachers - Trainee progress`